### DISCUSSION TITLE
Is there a way to use log-log axes for the degree distribution?

### DISCUSSION THREAD

**Discussion by williamjallen** [gephi/gephi#2637] [Q&A]
I often want to glance at the degree distribution of a graph I'm analyzing but it's hard to gain any meaningful insights when the distribution doesn't use a log-log scale.  Is there a way to change the axes, or do I just have to use another tool for this?

**Comment by mbastian** [comment]
No, unfortunately we don't offer that feature. But it's relevant so I suggest that you create a Wishlist issue and maybe describe a bit your use case in more details. We need to do an overhaul of the statistics reports at some point anyway so we shall consider this addition.
